source/common/filesystem/watcher_impl.cc

```diff
@@ -75,7 +77,7 @@ void WatcherImpl::onInotifyEvent() {
 
       for (FileWatch& watch : callback_map_[file_event->wd].watches_) {
         if (watch.file_ == file && (watch.events_ & events)) {
-          log_debug("matched callback: file: {}", file);
+          VLOG(1) << fmt::format("matched callback: file: {}", file);
```

Comment:
Given that log_debug is compiled out for non-debug builds, this should be DVLOG (here and elsewhere).
The same is true with log_trace also.